__tests__/app.test.js

```diff
@@ -76,3 +76,60 @@ describe(`/api/topics`, () => {
     });
   });
 });
+
+describe(`/api/articles/:article_id`, () => {
+  describe("GET", () => {
+    test("should return status code 200", () => {
+      return request(app).get("/api/articles/1").expect(200);
+    });
+
+    test("should respond with an array of articles", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .then(({ body }) => {
+          expect(Array.isArray(body.article)).toBe(true);
+        });
+    });
+
+    test("should respond with only the article matching the id", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .then(({ body }) => {
+          expect(body.article).toHaveLength(1);
+        });
+    });
+
+    test("should respond with an object containing the matched article with the correct properties", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .then(({ body: { article } }) => {
+          expect(article[0]).toBeInstanceOf(Object);
+          expect(article[0].article_id).toBe(1);
+          expect(article[0].title).toEqual(expect.any(String));
+          expect(article[0].author).toEqual(expect.any(String));
+          expect(article[0].body).toEqual(expect.any(String));
+          expect(article[0].topic).toEqual(expect.any(String));
+          expect(article[0].created_at).toEqual(expect.any(String));
+          expect(article[0].votes).toEqual(expect.any(Number));
+        });
+    });
+
+    test("should return a 404 error if path does not exist", () => {
+      return request(app)
+        .get("/api/articles/invalidpath")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Path not found!");
+        });
+    });
+
+    test("should return a 400 error if id is out of range or otherwise invalid", () => {
```

Comment:
While unlikely that a db has this many users, this number is a valid input for a user_id, should this be a 404?